torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -710,6 +712,28 @@ def pointless_permute_pair(match: Match, arg, perm1, perm2):
     match.erase_nodes()
 
 
+@register_graph_pattern(
+    CallFunction(
+        aten.bmm,
+        Arg(),
+        Arg(),
+    ),
+    pass_dict=patterns,
+)
+def bmm_to_mm(match: Match, mat1: torch.fx.Node, mat2: torch.fx.Node):
```

Comment:
We could also put this in the pytorch decompositions.. those get applied as we trace. either is fine.
 https://github.com/pytorch/pytorch/blob/a4c828199e05581481aae8e2273f84755ecb0be5/torch/_inductor/decomposition.py#L263-L286